packages/core/logging/core-logging-browser-internal/src/appenders/console_appender.ts

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import type { Layout, LogRecord, DisposableAppender } from '@kbn/logging';
+
+/**
+ *
+ * Appender that formats all the `LogRecord` instances it receives and logs them via built-in `console`.
+ * @internal
+ */
+export class ConsoleAppender implements DisposableAppender {
+  /**
+   * Creates ConsoleAppender instance.
+   * @param layout Instance of `Layout` sub-class responsible for `LogRecord` formatting.
+   */
+  constructor(private readonly layout: Layout) {}
+
+  /**
+   * Formats specified `record` and logs it via built-in `console`.
+   * @param record `LogRecord` instance to be logged.
+   */
+  public append(record: LogRecord) {
+    // eslint-disable-next-line no-console
+    console.log(this.layout.format(record));
```

Comment:
The MVP implements `.log` by default, which is fine to start with. 

In https://github.com/elastic/kibana/issues/144276 we'd have a bit more flexibility in how we approach a fuller feature set. Assuming we'd expand on wrapping the console and, since the specifics of how the console object depends on the browser, we should at least implement a subset of the de facto methods that log to std out:
`.error`, `.debug`, `.warn`,`.info`

Reply:
Yea. I didn't want to go into browser implementation details in the first version. I'll add a bullet point in https://github.com/elastic/kibana/issues/144276 to look at how it would be possible to have a proper browser console delegation using more methods